### DISCUSSION TITLE
Low Webview FPS slows down entire App

### DISCUSSION THREAD

**Discussion by CyberCyclone** [react-native-webview/react-native-webview#3596] [Q&A]
Hi,

I have a RN application that uses a 3D engine (BabylonJS) inside a Webview. When the Webview FPS drops (let's say from 60fps to 30fps), the UI FPS (as seen in the RN performance profiler) of the RN application drops to 30 as well. Is there a way to avoid this so that any performance issues inside the Webview don't impact the rest of the app?

Thanks!
Casey.

**Comment by caseygibson-emesent** [comment]
I've asked this here https://stackoverflow.com/questions/79143596/react-native-webview-performance-impacts-entire-app to hopefully find a solution.
